pandas/tests/util/test_deprecate_nonkeyword_arguments.py

```diff
@@ -44,6 +46,19 @@ def test_four_arguments():
         assert f(1, 2, 3, 4) == 10
 
 
+def test_three_arguments_with_name_in_warning():
+    with warnings.catch_warnings(record=True) as w:
+        warnings.simplefilter("always")
+        assert f(6, 3, 3) == 12
+        assert len(w) == 1
+        for actual_warning in w:
+            assert actual_warning.category == FutureWarning
+            assert str(actual_warning.message) == (
+                "Starting with pandas version 1.1 all arguments of f_add_inputs "
```

Comment:
Do you mean 1.5/2.0?

But wouldn't mind enforcing it immediately ;)

Reply:
this is just in a test of `deprecate_nonkeyword_arguments` itself